x-pack/plugins/apm/public/plugin.ts

```diff
@@ -71,21 +66,47 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
     this.initializerContext = initializerContext;
   }
   public setup(core: CoreSetup, plugins: ApmPluginSetupDeps) {
-    createCallApmApi(core.http);
     const config = this.initializerContext.config.get();
     const pluginSetupDeps = plugins;
 
     pluginSetupDeps.home.environment.update({ apmUi: true });
     pluginSetupDeps.home.featureCatalogue.register(featureCatalogueEntry);
 
     if (plugins.observability) {
+      const getApmDataHelper = async () => {
```

Comment:
should we wrap this in `once`, to cache it?

Reply:
this is cached by webpack be default we don't need to do that.